examples/lock-app/linux/README.md

```diff
@@ -15,3 +9,57 @@ The application could be build in the same manner as `all-clusters-app`:
 ```
 ? scripts/examples/gn_build_example.sh examples/lock-app/linux out/lock-app chip_config_network_layer_ble=false
 ```
+
+# Named Pipe Interface
+
+This application accepts commands over named pipe. Pipe is located in
+`/tmp/chip_lock_app_fifo-<PID>`.
+
+## Command Format
+
+Commands interface accepts commands formatted as a JSON object. Structure of the
+object:
+
+| Key      | Type   | Value Description                                                                               | Optional |
+| -------- | ------ | ----------------------------------------------------------------------------------------------- | -------- |
+| `Cmd`    | String | Contains name of the command to execute                                                         | No       |
```

Comment:
nit: Replace "Cmd" with "Name" to align with all-clusters-app

Reply:
Given where we are in the schedule, marking this as resolved. @Morozov-5F Please feel to follow up if you feel strongly.